src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -113,6 +113,11 @@ protected function assertWidgetMatchesXpath(FormView $view, array $vars, $xpath)
 
     abstract protected function renderLabel(FormView $view, $label = null, array $vars = array());
 
+    protected function renderHelp(FormView $view)
+    {
+        $this->markTestSkipped(sprintf('Legacy %s::renderHelp() is not implemented.', get_class($this)));
```

Comment:
let's remove the `Legacy ` prefix, it doesn't make sense, isn't it?

Reply:
I've just copy/paste. I don't really know what mean Legacy... I remove it. Then, I look at new errors that appears now in deps=high (deps=high do test with 3.4 ? Then I need to find a way to skip other tests too ?)